app/src/main/java/chat/rocket/android/layouthelper/chatroom/AbstractMessageViewHolder.java

```diff
@@ -14,6 +14,7 @@
 public abstract class AbstractMessageViewHolder extends ModelViewHolder<PairedMessage> {
   protected final RocketChatAvatar avatar;
   protected final ImageView userNotFoundAvatarImageView;
+  protected final ImageView imageViewSvgUserAvatar;
```

Comment:
Do we need to use a separate ImageView? This should probably be handled on RocketChatAvatar.